backend/services/queues/monthly_dump_queue_service.py

```
@@ -136,8 +147,26 @@ async def _process_message(self, message: Dict[str, Any], stats: Dict[str, int])
                 },
             )
 
-            if seed is None:
+            existing_dump_query = self.dump_controller.get({"id": monthly_dump_id}, maybe_single=True)
+            existing_dump = existing_dump_query.data if existing_dump_query else None
+
+            if existing_dump and existing_dump.get("status") == "completed":
+                logger.info(
```

Ruff (0.15.10)

[error] Too many arguments for `logging` format string

(PLE1205)